### PR TITLE
fix(mixed): fixed a couple of typos and added a todo

### PR DESCRIPTION
**Pull request by Veetaha**
Fixed a couple of typos and added a todo while studying the codebase.

### CODE REVIEW

**Comment by kjeremy** [comment]
Thanks!

bors r+

File: xtask/src/lib.rs [hunk 1]
```diff
@@ -53,6 +53,7 @@ fn reformat(text: impl std::fmt::Display) -> Result<String> {
     write!(rustfmt.stdin.take().unwrap(), "{}", text)?;
     let output = rustfmt.wait_with_output()?;
     let stdout = String::from_utf8(output.stdout)?;
+    // TODO: update the preable: replace ra_tools with the relevant path
```
Comment (edwin0cheng) [inline xtask/src/lib.rs]: We have a no todo check, but seem like ir do not parse xtask itself? 😃 

https://github.com/rust-analyzer/rust-analyzer/blob/cdcb3d3833d3d5b37b2cd4dac91a6e9366f20aea/xtask/tests/tidy-tests/cli.rs#L36

Comment (kiljacken) [reply xtask/src/lib.rs]: Seems like it.

@Veetaha We tend to use `// FIXME` for things like this, just for future use.

Comment (Veetaha) [reply xtask/src/lib.rs]: @edwin0cheng @kiljacken but why no to-dos? I can even see them all over the codebase

**Comment by matklad** [comment]
So that one can use `TODO` to make sure some *is* fixed before getting
merged. Ie, it is a feature that, if you have `TODO` comment, the CI will
fail, so you can actively use this to leave reminders for yourself when
working with big changes.

It can be argued that polarity is flipped, and that `FIXME` is the one that
should fail the CI, but here we just follow rust-lang/rust precedent.

**Pull request merged**